ansible/files/bootstrap/telemetry.sh

```diff
@@ -152,6 +152,12 @@ bash /mnt/$ANACONDA_SCRIPT -b -p $ANACONDA_PATH
 PIP_REQUIREMENTS_FILE=/tmp/requirements.txt
 aws s3 cp $TELEMETRY_CONF_BUCKET/bootstrap/python-requirements.txt $PIP_REQUIREMENTS_FILE
 $ANACONDA_PATH/bin/pip install -r $PIP_REQUIREMENTS_FILE
+
+conda create -n zeppelin python=3.6 cairo pillow -y -q
+source activate zeppelin
+pip install python-zeppelin
```

Comment:
Python-zeppelin requires the machine to be on Python 3.6. We can create another file called `python3-requirements.txt` and have `pip install python-zeppelin` reside in there. Or we can migrate everything to 3.6.